snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -3103,7 +3053,7 @@ static void AppendEx()
             Console.WriteLine(string.Join(", ", numbers.Append(5)));
 
             // If you prefer, you can create a new list explicitly
-            List<int> newNumbers = numbers.Append(5).ToList();
+            List<int> newNumbers = [.. numbers, 5];

```

Comment:
There are a lot of positive impacts of collection expressions, but the purpose of this snippet is to demonstrate the `Append` method. Maybe something like this:

```suggestion
            List<int> newNumbers = [.. numbers.Append(5)];

```